src/components/sectionsWheelPicker/SectionsWheelPicker.driver.tsx

```diff
@@ -1,21 +1,35 @@
-import _ from 'lodash';
+import React from 'react';
 import {useComponentDriver, ComponentProps} from '../../testkit/new/Component.driver';
 import {WheelPickerDriver} from '../WheelPicker/WheelPicker.driver';
 import {SectionsWheelPickerProps} from './index';
 
-export const SectionsWheelPickerDriver = (props: ComponentProps) => {
-  const driver = useComponentDriver<SectionsWheelPickerProps>(props);
-  const sections = driver.getProps().children as SectionsWheelPickerProps;
-  const sectionsDrivers = _.map(sections, (_, index) => {
-    const sectionTestID = `${props.testID}.${index}`;
-    return WheelPickerDriver({
-      renderTree: props.renderTree,
-      testID: sectionTestID
-    });
-  });
 
-  
+type CustomSectionsTestIds = {
+  customSectionTestIds?: string[]
+};
+
+export const SectionsWheelPickerDriver = (props: ComponentProps & CustomSectionsTestIds) => {
+  const {customSectionTestIds, ...others} = props;
+  const driver = useComponentDriver<SectionsWheelPickerProps>(others);
 
+  let sectionsDrivers: (ReturnType<typeof WheelPickerDriver>)[];
+  const {renderTree} = props;
+  if (customSectionTestIds) {
+    sectionsDrivers = customSectionTestIds.map(testID => {
```

Comment:
When iterating over the children we can't get the testID from each prop? The testID is passed to the section (wheel picker) container, no?

Reply:
We could but for the same reason above.